k8s-oci.tf

```diff
@@ -471,9 +470,15 @@ module "k8smaster-public-lb" {
   shape                     = "${var.k8sMasterLBShape}"
 }
 
+module "nginx-lb" {
+  source        = "./network/loadbalancers/nginx"
+  hosts         = "${concat(module.instances-k8smaster-ad1.private_ips,module.instances-k8smaster-ad2.private_ips, module.instances-k8smaster-ad3.private_ips)}"
+  nginx_version = "${var.nginx_version}"
+}
+
 module "kubeconfig" {
   source                     = "./kubernetes/kubeconfig"
   api_server_private_key_pem = "${module.k8s-tls.api_server_private_key_pem}"
   api_server_cert_pem        = "${module.k8s-tls.api_server_cert_pem}"
-  k8s_master                 = "https://${module.k8smaster-public-lb.ip_addresses[0]}:443"
+  k8s_master                 = "{local.master_lb_address}"
```

Comment:
There's still a typo here. `"{local.master_lb_address}"` is an invalid value. I am sure you meant `"${local.master_address}"` (note the missing `$`)

Beyond the typo, when `master_lb_enabled = false` and `control_plane_subnet_access = "public"`, we want the value of `k8s_master` to be based on `master_public_ip` or a `worker_public_ip`.

The purpose of the kubeconfig module is to generate the file `generated/kubeconfig` for the person running Terraform to use.  `https://127.0.0.1:6443` is not the right value when `master_lb_enabled = false` and `control_plane_subnet_access = "public"`.